distributions/Lakka/options

```diff
@@ -435,18 +435,9 @@ DISTRO_SRC="http://sources.libreelec.tv/$LIBREELEC_VERSION"
       RA_PLAYLIST_CORES="${RA_PLAYLIST_CORES//genesis_plus_gx/picodrive}"
     fi
   elif [ "$PROJECT" = "Amlogic" ]; then
-    if [ "$DEVICE" != "S912" ]; then
-      RA_PLAYLIST_CORES="${RA_PLAYLIST_CORES//parallel_n64/mupen64plus_next}"
-    fi
-  fi
-  if [ "$PROJECT" = "WeTek_Core" -o "$PROJECT" = "WeTek_Play" ]; then
     RA_PLAYLIST_CORES="${RA_PLAYLIST_CORES//parallel_n64/mupen64plus_next}"
   fi
 
-  if [ "$PROJECT" != "Generic" -a "$DEVICE" != "RPi" ]; then
-    RA_PLAYLIST_CORES="${RA_PLAYLIST_CORES//mame/mame2003-plus}"
-  fi
-
```

Comment:
I do not even understand how this even worked - this would replace:
mame_libretro.so => mame2003-plus_libretro.so
mame2003_plus_libretro.so => mame2003-plus2003_libretro.so
mame2010_libretro.so => mame2003-plus2010_libretro.so
mame2015_libretro.so => mame2003-plus2015_libretro.so
mame_libretro.so => mame2003-plus_libretro.so

which are nonexisten files.